observability/logging/cluster-logging-support.adoc

```diff
@@ -0,0 +1,73 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cluster-logging-support"]
+include::_attributes/common-attributes.adoc[]
+= Support
+:context: cluster-logging-support
+
+toc::[]
+
+include::snippets/logging-supported-config-snip.adoc[]
+include::snippets/logging-compatibility-snip.adoc[]
+include::snippets/log6x-loki-statement-snip.adoc[]
+
+{logging-uc} {for} is an opinionated collector and normalizer of application, infrastructure, and audit logs. You can use it to forward logs to various supported systems.
+
+{logging-uc} is not:
+
+* A high scale log collection system
+* Security Information and Event Monitoring (SIEM) compliant
+* A "bring your own" (BYO) log collector configuration
+* Historical or long term log retention or storage
+* A guaranteed log sink
+* Secure storage - audit logs are not stored by default
+
+[id="cluster-logging-support-CRDs_{context}"]
+== Supported API custom resource definitions
+
+The following table describes the supported {logging-uc} APIs.
+
+.Loki API support states
+[cols="3",options="header"]
+|===
+|CustomResourceDefinition (CRD)
+|ApiVersion
```

Comment:
I recognize that this probably refers to some `APIVersion` parameter in YAML files, but personally it feels weird to neither have this as two words or have it in backticks. Do you think it would still make sense either as "API version" or in backticks like `APIVersion`? 

I feel like it should be changed but I will defer to your judgement here

Reply:
added backticks